### ISSUE TITLE
Support for Symfony/PHP sessions

### ISSUE THREAD

**Issue by Gemorroj** [k911/swoole-bundle#6]
``` bin/console swoole:server:run


 [OK] Swoole HTTP Server started on http://localhost:9501


 ----------------- -------------------------
  Configuration     Values
 ----------------- -------------------------
  env               dev
  debug             true
  worker_count      4
  memory_limit      512 MiB
  trusted_hosts     127.0.0.1, localhost
  trusted_proxies   *
  public_dir        /var/www/wapinet/public
 ----------------- -------------------------

22:50:53 CRITICAL  [request] Uncaught PHP Exception RuntimeException: "Failed to start the session because headers have already been sent by "" at line 0." at /var/www/wapinet/vendor/symfony/http-foundation/Session/Storage/NativeSessionStorage.php line 138 ["exception" => RuntimeException { …}] []
22:50:53 CRITICAL  [request] Uncaught PHP Exception RuntimeException: "Failed to start the session because headers have already been sent by "" at line 0." at /var/www/wapinet/vendor/symfony/http-foundation/Session/Storage/NativeSessionStorage.php line 138 ["exception" => RuntimeException { …}] []
22:51:18 CRITICAL  [request] Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Failed to start the session because headers have already been sent by "" at line 0.")." at /var/www/wapinet/vendor/symfony/web-profiler-bundle/Resources/views/Profiler/layout.html.twig line 107 ["exception" => Twig_Error_Runtime { …}] []
```

**Comment by k911** [comment]
I think it is swoole related issue. 
I've fixed it by using this configuration:

```
framework:
   session:
        storage_id: session.storage.php_bridge
```

**Comment by Gemorroj** [comment]
https://github.com/swoole/swoole-src/issues/1828

**Comment by k911** [comment]
Fixed by #73
